project.json

```
@@ -4,7 +4,7 @@
   "// These targets are used for repo level utils and checking repo files which do not belong to specific published packages": {},
   "targets": {
     "typecheck": {
-      "command": "tsc"
+      "command": "tsc -c tsconfig.repo-config.files.json"
```

Just 
```suggestion
      "command": "tsc -b --emitDeclarationOnly"
```
works as well but I think this does extra work by typechecking all the descendants in the whole repo, not just the root-owned TS.